libs/clients/driving-license-book/src/lib/drivingLicenseBookClient.service.ts

```diff
@@ -335,4 +338,28 @@ export class DrivingLicenseBookClientApiFactory {
     })
     return data?.bookId || null
   }
+
+  private async hasPracticeDriving(id: string): Promise<boolean> {
+    const api = await this.create()
+    const { data } = await api.apiStudentGetLicenseBookIdGet({ id })
+    return data?.practiceDriving || false
+  }
+
+  async allowPracticeDriving({
+    teacherNationalId,
+    studentNationalId,
+  }: AllowedPractieDrivingInput) {
+    const api = await this.create()
+    try {
+      await api.apiTeacherCreateAllowedPracticeDrivingPost({
+        createAllowedPractieDrivingRequestBody: {
+          teacherSsn: teacherNationalId,
+          studentSsn: studentNationalId,
+        },
+      })
+      return { success: true }
```

Comment:
Less legible, but 4 lines shorter :sunglasses: 
```suggestion
      return { success: !!await api.apiTeacherCreateAllowedPracticeDrivingPost({
        createAllowedPractieDrivingRequestBody: {
          teacherSsn: teacherNationalId,
          studentSsn: studentNationalId,
        },
      }).catch(return false)}
```